app/code/Magento/Backend/Model/Search/ItemsAbstract.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Model\Search;
+
+use Magento\Backend\Api\Search\ItemsInterface;
+use Magento\Framework\DataObject;
+
+abstract class ItemsAbstract extends DataObject implements ItemsInterface
```

Comment:
introducing of abstract classes is a bad practice. In M2 we are trying to avoid using Inheritance Based API.

use composition instead, if your main reason for Inheritance usage to get rid of Copy-Paste